widget/form.go

```diff
@@ -119,6 +124,25 @@ func (f *Form) Disabled() bool {
 	return f.disabled
 }
 
+// SetOnValidationChanged is intended for parent widgets or containers to hook into the validation.
```

Comment:
Sorry for not realising this, but should we not have the `// The function might be overwritten by a parent that cares about child validation (e.g. widget.Form).` comment that the entry also has?

Reply:
I feel like a case where it would be overwritten (by say a parent form) is a bit rare, but I don't think there's a good reason why not.

Reply:
I think it makes sense to at least indicate that it might be overwritten. The function is the same one as in entry and that one has the same comment.